docs/source/configuration/slots.md

```diff
@@ -0,0 +1,260 @@
+---
+myst:
+  html_meta:
+    "description": "Slots are insertion points in the Volto rendering tree structure."
+    "property=og:description": "Slots are insertion points in the Volto rendering tree structure."
+    "property=og:title": "Slots"
+    "keywords": "Volto, Plone, frontend, React, configuration, slots, viewlets"
+---
+
+# Slots
+
+Slots are insertion points in the Volto rendering tree structure.
+This concept is inherited from the Plone Classic UI {doc}`plone:classic-ui/viewlets`.
+
+
+## Anatomy
+
+Slots are named, and they can contain a list of different slot components.
+Slot components are also named, and they are registered in the configuration registry using a specific API for slots.
```

Comment:
Link to the specific API instead of just hand-waving at it.

Reply:
Does the configuration registry API for slots exist yet? https://6.docs.plone.org/volto/configuration/how-to.html

Reply:
@stevepiercy It's currently discussed further down in this file: https://github.com/plone/volto/blob/33b96c9d3e6d1636bc7e6c0224b5b86df61bcf28/docs/source/configuration/slots.md?plain=1#L65

Reply:
Fixed in [049e16a](https://github.com/plone/volto/pull/5775/commits/049e16a846405392473e99ea08e245e407391443)